t/core/config_etcd.t

```diff
@@ -118,42 +118,7 @@ qr/(closed){1,}/
 
 
 
-=== TEST 4: originate TLS connection to etcd cluster and verify TLS certificate (default behavior)
```

Comment:
Now we should make a CA bundle into the https://github.com/openresty/lua-nginx-module#lua_ssl_trusted_certificate directive in "/" location, instead of the conf server one.